### PR TITLE
feat: add fullscreen `CCardDialog`

### PR DESCRIPTION
**Pull request by ItsPierre**
Add fullscreen `CCardDialog`.
Add property `fullscreen` to `CCardDialog` and `CCard`-`mode` `fullscreen`.

### CODE REVIEW

File: src/components/CCard.vue [hunk 1]
```diff
@@ -200,6 +200,11 @@ const props = withDefaults(
      * This also disables all buttons including the close-button, if shown.
      */
     disabled?: boolean;
+
+    /**
+     * Changes layout for fullscreen display (primarily in dialog).
+     */
+    fullscreen?: boolean;
```
Comment (johannes-huther) [inline src/components/CCard.vue]: I think this is redundant?

File: src/components/CCard.vue [hunk 2]
```diff
@@ -236,6 +241,8 @@ const cardClass = computed<string>(() => {
     classes += " pb-6 pb-md-8";
   }
 
+  if (props.fullscreen) classes += " ma-0";
+
```
Comment (johannes-huther) [inline src/components/CCard.vue]: should check mode===fullscreen?

File: src/components/CCard.vue [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <template>
   <v-card
-    rounded="xl"
+    :rounded="mode == 'fullscreen' ? false : 'xl'"
```
Comment (johannes-huther) [inline src/components/CCard.vue]: ===

**Review by johannes-huther — CHANGES_REQUESTED** [review]

File: src/components/CCard.vue (hunk 2 above)
Comment (ItsPierre) [reply src/components/CCard.vue]: forgot to remove this

**Comment by ItsPierre** [comment]
https://github.com/Crystal-Creations-GbR/crystal-components/compare/169cf3719f404104be92c7eb15a58f23b59ad2d1..8ffa94066b2936a5fd3a8828b760cd9ea21acc42

**Review by johannes-huther — APPROVED** [review]

**Pull request merged**